### ISSUE TITLE
Request to update the survey field notes datasets published from NHMO

### ISSUE THREAD

**Issue by dagendresen** [gbif-norway/helpdesk#47]
Request from Bjørn Petter to update the _survey_ | _field diary_ | _field notes_ datasets ("_kryssliste_") published by NHMO.

Vascular plants survey observations: https://doi.org/10.15468/w8gru5 (updated 2019-12-06)
Fungi survey observations: https://doi.org/10.15468/dx42gy (updated 2019-12-06)
Lichen field notes: https://doi.org/10.15468/zrfxcu (updated 2020-01-06)
Maybe this dataset also? https://doi.org/10.15468/tvnjk7 (updated 2019-10-29)

Are these pages with information on the field diary notes still up to date and relevant?
Fungi field notes: http://www.nhm2.uio.no/botanisk/sopp/xlist.htm
Lichen field notes: http://nhm2.uio.no/botanisk/lav/XLIST.HTM

For vascular plants field notes - see also: 
https://www.muspro.uio.no/fag/botanikk/karplanter2.shtml
https://www.nhm.uio.no/forskning/samlinger/botanikk/karplanteherbariet/

Where does the source data come from? Are the source data archived (stored?) somewhere that the IPT can fetch them from? Preferably as an automatic (daily?, weekly?, monthly?) routine?

**Comment by rukayaj** [comment]
Vascular plants field notes: 
https://doi.org/10.15468/w8gru5 (updated 2019-12-06)
https://ipt.gbif.no/manage/resource.do?r=o_vxl
4 .txt files, xaa, xab, xac, xad

Fungi field notes: 
https://doi.org/10.15468/dx42gy (updated 2019-12-06)
https://ipt.gbif.no/manage/resource?r=nxs
Local mariadb database - nxs [sql] | db=NXS

Lichen field notes: 
https://doi.org/10.15468/zrfxcu (updated 2020-01-06)
https://ipt.gbif.no/manage/resource?r=nxl
1 occurrence .txt file

Maybe Vascular Plants, Observations, Oslo (O) UiO also? 
https://doi.org/10.15468/tvnjk7 (updated 2019-10-29)
https://ipt.gbif.no/manage/resource?r=o_v_gps
Local mariadb database - o_v_gps [sql] | db= O_V_GPS

So the source data are a mix, either local maria db or uploaded text files. I have no idea where they came from originally but I see you have asked Bjørn Petter.

**Comment by dagendresen** [comment]
I think that the MariaDB was only something Christian used (for data quality testing routines and as the mechanism to trigger the IPT to reindex daily, weekly, etc) and thus not the source data. I think vascular plant field notes source data was provided (as manual data export) from Oddvar Pedersen and Lichen field notes from Einar Timdal and fungus field notes source data maybe from Anne Molina (?). [@Bjørn Petter?]

**Comment by dagendresen** [comment]
Meeting with Olav and Oddvar on the "krysslister"

When:	Wed 2021-09-15 10:00-12:00
Where:	Tøyen: Lids - Konferanserom K29 (12)

**Comment by rukayaj** [comment]
This meeting was postponed

**Comment by dagendresen** [comment]
Update May 2024:
Bjørn Petter writes that: Reidar Elven har krysslister som inneholder drøye 94 000 observasjoner. Dette er punchede data fra hans krysslister fra 1980 og ca. 2020. Ønsker disse integrert i eksisterende krysslister

Vascular plants field notes: https://doi.org/10.15468/w8gru5 

Additional request to update t[he field notes portal](https://www.muspro.uio.no/fag/botanikk/karplanter.shtml) (...) which was NOT developed by the GBIF-node and likely very hard to update from the GBIF node --> maybe possible to explore creating a hosted portal or something if useful...?

**Comment by rukayaj** [comment]
Yes i agree, definitely recommend a hosted portal for this! Or maybe Eirik should add it to https://samlingsportal.nhm.uio.no/museum/nhm/

**Comment by Rindiser** [comment]
I think the most important thing is to have a system for updating / correcting the data. As I remember Mikael have gotten the original dBase files for Oddvar so that he can transform them into a useful format. If we can get all the relevant information on the IPT then that would be a great first step. Make a portal to host the data might not be completely in line with our strategy, but that is something we can discuss.
